concourse/tasks/update-task-definition.yml

```diff
@@ -0,0 +1,38 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/terraform
+    tag: terraform-0.13.3
+inputs:
+  - name: govuk-infrastructure
+    path: src
+  - name: release
+outputs:
+  - name: terraform-outputs
+params:
+  AWS_REGION: eu-west-1
```

Comment:
Might be nice to pass this into Terraform (but definitely outside the scope of this PR)